components/checkbox/index.css

```diff
@@ -20,15 +20,16 @@
 	--spectrum-checkbox-checkmark-color: var(--highcontrast-checkbox-background-color-default, var(--mod-checkbox-checkmark-color, var(--spectrum-gray-50)));
```

Comment:
The pattern used on this line doesn't seem to be consistent with where we typically define the mods and highcontrast variables. This seems like a good time to clean this up.